# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/bip/hotfix-cmp.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- bip-0.9.3.orig/src/util.h	2022-03-10 14:21:01.000000000 +0100
+++ bip-0.9.3/src/util.h	2026-07-26 06:30:45.044132927 +0200
@@ -48,7 +48,7 @@
 typedef struct list {
 	struct list_item *first;
 	struct list_item *last;
-	int (*cmp)();
+	int (*cmp)(const void *, const void *);
 } list_t;
 #pragma GCC diagnostic pop
